frontend/src/pages/Resources/components/ResourceSharingForm/ResourceSharingForm.jsx

```diff
@@ -82,6 +93,44 @@ export function ResourceSharingForm(props) {
     setFormErrors(errors);
   };
 
+  const submitResourceFormData = async (formdata) => {
+    var api = `${END_POINT}/resources/`;
+    setIsLoading(true);
+    return await fetch(api, {
+      method: "POST",
+      body: JSON.stringify(formdata),
+      headers: {
+        "Content-type": "application/json",
+      },
+    })
+      .then((res) => res.json())
+      .then((data) => {
+        if (
+          data.message === "Database Error" ||
+          data.message === "Sendgrid Error"
+        ) {
+          setIsLoading(false);
+          setResourceToast(data.message);
+          setResourceToastStatus("error");
+        } else {
+          setIsLoading(false);
```

Comment:
Can we please show the loader on the whole page while the submission happens instead of just one section of the page

Reply:
Hello @Kajol-Kumari, currently the resource page is designed and working as per the theme and scenario of the contact page. The loader in resource page is working exactly like contact page. 
If still I have to make changes in it, then please inform.

Reply:
please feel free to raise an issue for this and the new pr with the change as we need consistency everywhere